constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -200,20 +200,12 @@ func (d *Driver) Query(ctx context.Context, target string, constraints []*unstru
 		// TODO: should namespace be made available, if possible? Generally that context should be present
 		response := validator.Validate(ctx, versionedAttr.GetResource(), versionedAttr, constraint, nil, celAPI.PerCallLimit, nil)
 
-		enforcementAction, found, err := unstructured.NestedString(constraint.Object, "spec", "enforcementAction")
-		if err != nil {
-			return nil, err
-		}
-		if !found {
-			enforcementAction = apiconstraints.EnforcementActionDeny
```

Comment:
We should not remove this for non-scope EA, this is still needed for backward compat.

Reply:
enforementAction and scopedEnforcementActions are set [here ](https://github.com/open-policy-agent/frameworks/pull/403/files/2428b2195db8b85fd45f55d16fc1559c2ad00203#diff-2b54391e78f46b40ac71789b492e315a83cbd6b15716c000dcd4c5915d907802R741)- once driver is done with query and returns responses.